ios/Sources/Component/Card/MediumCard.swift

```diff
@@ -0,0 +1,111 @@
+        self.tapFavoriteAction = tapFavoriteAction
+    }
+
+    public var body: some View {
+        Button(action: tapAction, label: {
+            VStack(alignment: .leading, spacing: 16) {
+                VStack(alignment: .leading, spacing: 13) {
+                    ImageView(imageURL: imageURL, width: 225, height: 114)
+
+                    VStack(alignment: .leading, spacing: 12) {
+                        VStack(alignment: .leading, spacing: 4) {
+                            Text(title)
+                                .font(.subheadline)
+                                .foregroundColor(Color(AssetColor.Base.primary.color))
+                                .lineLimit(2)
+
+                            Text(date.formatted)
+                                .font(.caption)
+                                .foregroundColor(Color(AssetColor.Base.tertiary.color))
+                        }
+
+                        HStack(spacing: 8) {
+                            Tag(type: tag) {
+                                // do something if needed
+                            }
+
+                            Spacer()
+
+                            Button(action: tapFavoriteAction, label: {
+                                if isFavorited {
+                                    Image(uiImage: AssetImage.iconFavorite.image.withRenderingMode(.alwaysTemplate).withTintColor(AssetColor.primary.color))
+                                } else {
+                                    Image(uiImage: AssetImage.iconFavoriteOff.image.withRenderingMode(.alwaysTemplate).withTintColor(AssetColor.primary.color))
+                                }
+                            })
+                        }
+                    }
+                }
+            }
+            .padding(16)
+        })
+    }
+}
+
+struct MediumCard_Previews: PreviewProvider {
+    static var previews: some View {
+        Group {
+            MediumCard(
+                title: "タイトルタイトルタイトルタイトルタイタイトルタイトルタイトルタイトルタイト...",
+                imageURL: URL(string: ""),
+                tag: .droidKaigiFm,
+                date: Date(),
+                isFavorited: false,
+                tapAction: {},
+                tapFavoriteAction: {}
+            )
+            .frame(width: 257, height: 258)
+
+            MediumCard(
+                title: "タイトルタイトルタイトルタイトルタイタイトルタイトルタイトルタイトルタイト...",
+                imageURL: URL(string: ""),
+                tag: .medium,
+                date: Date(),
+                isFavorited: true,
+                tapAction: {},
+                tapFavoriteAction: {}
+            )
+            .frame(width: 257, height: 258)
+
+            MediumCard(
+                title: "タイトル",
+                imageURL: URL(string: ""),
+                tag: .youtube,
+                date: Date(),
+                isFavorited: true,
+                tapAction: {},
+                tapFavoriteAction: {}
+            )
+            .frame(width: 257, height: 258)
```

Comment:
Is this for confirming `tag` previews?
If so, I think `ForEach` is useful. 😄

Reply:
Not only tags, also title length is changing so, I want leave this as it is/ 
Anyway, I will add darkmode preview as well.